### ISSUE TITLE
Ways To Give - Mail

### ISSUE THREAD

**Issue by etrepum** [MissionBit/missionbit.org#14]
[Fundraise Page Language](https://docs.google.com/document/d/1u8DQEzqyiEisfDJYvH0WM1Tc1vCjogP7pv9Pdx3Qbng/edit)

By Mail

> Please fill out our Donation Form and address all incoming mail to:
> Mission Bit
> 101A Clay St., #121 
> San Francisco, CA 94111

Blocked on: Donation Form

**Comment by etrepum** [comment]
This is superseded by the new design
